predicthq/endpoints/decorators.py

```diff
@@ -16,16 +16,25 @@ def _to_url_params(data, glue=".", separator=","):
         elif isinstance(value, list):
             params[key] = separator.join(map(str, value))
         elif isinstance(value, dict):
-            for subkey, subvalue in value.items():
-                if isinstance(subvalue, list):
-                    params[glue.join((key, subkey))] = separator.join(map(str, subvalue))
-                else:
-                    params[glue.join((key, subkey))] = subvalue
+            params.update(**_flatten_dict(value, glue, separator, pk=key))
```

Comment:
Why do you use argument unpacking here but not on line 29? should we stick to the same call type for consistency?

Reply:
Good catch, updating now.